sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/InstrumentationRegistryHelper.java

```diff
@@ -6,22 +6,40 @@
 package com.microsoft.appcenter.utils;
 
 import android.os.Bundle;
-import android.support.test.InstrumentationRegistry;
+
+import java.lang.reflect.Method;
 
 /**
  * Wraps InstrumentationRegistry class to enable mocking in unit tests.
  */
 public class InstrumentationRegistryHelper {
 
+    private static final String[] LOCATIONS = new String[]{"androidx.test.platform.app.InstrumentationRegistry",
+            "androidx.test.InstrumentationRegistry",
+            "android.support.test.InstrumentationRegistry"};
+
     /**
      * Get the instrumentation arguments from the InstrumentationRegistry. Wrapper exists for unit
      * tests.
      *
      * @return the instrumentation arguments.
-     * @throws LinkageError          if the class, method is not found or does not match, typically no test dependencies in release.
-     * @throws IllegalStateException if no argument Bundle has been registered.
+     * @throws IllegalStateException if it cannot call getArguments using any of the provided classes.
      */
-    public static Bundle getArguments() throws LinkageError, IllegalStateException {
-        return InstrumentationRegistry.getArguments();
+    public static Bundle getArguments() throws IllegalStateException {
+        Exception exception = null;
+        for (String location : LOCATIONS) {
+            try {
+                Class<?> aClass = getClass(location);
+                Method getArguments = aClass.getMethod("getArguments", (Class[]) null);
+                return (Bundle) getArguments.invoke(null, (Object[]) null);
```

Comment:
Here too, remove the second parameter.

Reply:
Done - sorry, I was sure I'd done these!